test/views/declaration/summary/SummaryPageViewSpec.scala

```diff
@@ -216,4 +231,28 @@ class SummaryPageViewSpec extends WordSpec with MustMatchers with ExportsDeclara
       amendSummaryPage must include("summary.amend.information")
     }
   }
+
+  private def createDeclaration(declarationType: DeclarationType = DeclarationType.STANDARD): ExportsDeclaration = {
+    val declaration = aDeclaration(
+      withType(declarationType),
+      withConsignmentReferences(),
+      withDestinationCountries(),
+      withGoodsLocation(GoodsLocation("PL", "type", "id", Some("a"), Some("b"), Some("c"), Some("d"), Some("e"))),
+      withWarehouseIdentification(Some("a"), Some("b"), Some("c"), Some("d")),
+      withOfficeOfExit("id", Some("code")),
+      withContainerData(Container("id", Seq.empty)),
+      withTotalNumberOfItems(Some("123"), Some("123")),
+      withNatureOfTransaction("nature"),
+      withItem(anItem())
+    )
+    declaration
+  }
+
+  private def createJourneyRequest(declaration: ExportsDeclaration) =
```

Comment:
I am preety sure that one of BaseSpec bash similar helper - maybe UnitViewSpec

Reply:
This will be addressed in a ticket that will revamp the Check your answers page.